src/test/java/com/glovoapp/planningpoker/e2e/voting/DanceTest.java

```diff
@@ -0,0 +1,45 @@
+package com.glovoapp.planningpoker.e2e.voting;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import com.glovoapp.planningpoker.e2e.pages.SessionPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.Optional;
+
+public class DanceTest {
+
+    private final WebDriver driver = getDriver();
+    private MainPage mainPage = new MainPage(driver);
+    private SessionPage sessionPage = new SessionPage(driver);
+    private final String userName = "username";
```

Comment:
Constant should be `static`.

```suggestion
    private static final String USERNAME = "username";
```